daemon/emer-main.c

```diff
@@ -414,8 +419,13 @@ make_daemon (gint                argc,
     permissions_provider =
       emer_permissions_provider_new_full (config_file_path, NULL);
 
+  if (tracking_id_file_path != NULL)
+    machine_id_prov =
+      emer_machine_id_provider_new_full(tracking_id_file_path);
```

Comment:
Where is this function implemented?

Reply:
It's in daemon/emer-machine-id-provider.c line 171.

Reply:
Sorry, my bad, I thought you were introducing this new class.